packages/babel-parser/src/util/location.ts

```diff
@@ -6,14 +6,15 @@ export type Pos = {
 // `startLoc` and `endLoc` properties.
 
 export class Position {
+  // If `declare` is used here, the `jquery` benchmark will drop by 10%. :)
```

Comment:
Did you mean if `declare` is used for `line` and `column`? Since the `index` property is declared.

Reply:
Yes, if I use `declare` for `line` and `column`, the performance will decrease.🤦‍♂️